### PR TITLE
LPS-174964 | Add Automation Test

### PR DESCRIPTION
**Pull request by andreluis2022**
Ticket:
https://issues.liferay.com/browse/LPS-174964

Description:
column Date Modified is present and
shows label "Workspace" is present

ClientExtensionUsingConfigFile#OsgiClientExtensionWorkspaceLabelIsPresent

### CODE REVIEW

**Comment by andreluis2022** [comment]
@mateusralv

**Comment by john-co** [comment]
Just started reviewing :)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtensionUsingConfigFile.testcase [hunk 1]
```diff
@@ -88,9 +88,24 @@ definition {
...
+				category = "Custom Apps",
+				panel = "Applications",
+				portlet = "Client Extensions");
+		}
 
-		// TODO LPS-171621 OsgiClientExtensionWorkspaceLabelIsPresent pending implementation
+		task ("Then (date column) and label (workspace) are present") {
+			AssertTextPresent(value1 = "Modified Date");
 
+			AssertTextPresent(value1 = "Workspace");
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtensionUsingConfigFile.testcase:107]: You need to use proper xpath locators, having only value1 is not enough for AssertTextPresent.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtensionUsingConfigFile.testcase [hunk 2]
```diff
@@ -88,9 +88,24 @@ definition {
 	@description = "Verify that the Workspace label is present when a client extension is created through osgi"
 	@priority = 3
 	test OsgiClientExtensionWorkspaceLabelIsPresent {
+		task ("Given config file for client extension of type custom element created") {
+			ClientExtension.createGlobalVariableClientExtensionType(type = "customElement");
+
+			ClientExtension.checkConfigFileCreatedProperly();
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtensionUsingConfigFile.testcase:94]: Remove this line. We don't want assertions in Given and When steps, we only want to make validation checks on what we actually want to test in the Then steps.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtensionUsingConfigFile.testcase (hunk 1 above)
Comment (mateusralv) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtensionUsingConfigFile.testcase:107]: Or could you use `FDSCustomView.assertColumnPresent(key_field = "Modified Date");` for asserts.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtensionUsingConfigFile.testcase [hunk 3]
```diff
@@ -88,9 +88,24 @@ definition {
...
+		}
+
+		task ("When Go to applications/custom apps/client extensions") {
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtensionUsingConfigFile.testcase:97]: Copy the Given When Then from the ticket: https://issues.liferay.com/browse/LPS-174964

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtensionUsingConfigFile.testcase [hunk 4]
```diff
@@ -88,9 +88,24 @@ definition {
...
+			ApplicationsMenu.gotoPortlet(
+				category = "Custom Apps",
+				panel = "Applications",
+				portlet = "Client Extensions");
+		}
 
-		// TODO LPS-171621 OsgiClientExtensionWorkspaceLabelIsPresent pending implementation
+		task ("Then (date column) and label (workspace) are present") {
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtensionUsingConfigFile.testcase:104]: Copy the Given When Then from the ticket: https://issues.liferay.com/browse/LPS-174964

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtensionUsingConfigFile.testcase (hunk 1 above)
Comment (john-co) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtensionUsingConfigFile.testcase:107]: I think the same macros here will work: https://github.com/liferay-frontend/liferay-portal/pull/3039#issuecomment-1426222861

Comment (andreluis2022) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtensionUsingConfigFile.testcase:107]: This tip is important!!
thanks John

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by mateusralv** [comment]
Hi @andreluis2022 You could solve the rebase problems and remove those `. java`files? don’t seem related to your test.

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by andreluis2022** [comment]
@mateusralv

**Comment by mateusralv** [comment]
ci:test:frontend

**Comment by mateusralv** [comment]
ci:test:frontend

**Comment by mateusralv** [comment]
I just change the test file and the test continues passing on CI 🎉 Seems good to me forward, thanks @andreluis2022!

**Comment by mateusralv** [comment]
ci:test:sf

**Comment by mateusralv** [comment]
ci:foward

**Pull request closed without merging**